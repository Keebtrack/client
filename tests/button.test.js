import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../src/components/button';


test('Button renders', () => {
  const component = renderer.create(<Button onClickHandler={ () => null } />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  tree.props.onClick();
  expect(tree).toMatchSnapshot();
});