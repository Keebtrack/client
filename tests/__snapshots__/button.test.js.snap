// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders 1`] = `
<button
  onClick={[Function]}
>
  click me!
</button>
`;

exports[`Button renders 2`] = `
<button
  onClick={[Function]}
>
  click me!
</button>
`;
